### ISSUE TITLE
key error on yt.title

### ISSUE THREAD

**Issue by fedetony** [JuanBindez/pytubefix#256]
**:exclamation:  DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE :exclamation:**

lack of information will lead to closure of the issue

__________
**Describe the bug**
when running extract.py gets a key error:
` File "C:\Users\Tony\AppData\Roaming\Python\Python39\site-packages\pytubefix\__main__.py", line 693, in title
    self.check_availability()
  File "C:\Users\Tony\AppData\Roaming\Python\Python39\site-packages\pytubefix\__main__.py", line 312, in check_availability
    status, messages = extract.playability_status(self.vid_info)
  File "C:\Users\Tony\AppData\Roaming\Python\Python39\site-packages\pytubefix\extract.py", line 110, in playability_status
    if 'isLive' in player_response['videoDetails']:
KeyError: 'videoDetails'`

__________
**code that was used that resulted in the bug**
error happens while reading: yt.title
for any video url I give.
```python
from pytubefix import YouTube
def get_yt_video_from_url(self,url,client='WEB_CREATOR'):
        try:
            yt = YouTube(url, client=client, on_progress_callback = self._on_progress)
        except Exception as eee:
            #print(eee)
            self.to_log.emit("Error Video: {}".format(eee))
            yt = None
        return yt

def get_url_info(self,url):
        yt_info={}
        yt = self.get_yt_video_from_url(url)
        if yt:
            yt_info.update({"title":yt.title})
            yt_info.update({"age_restricted":yt.age_restricted})
            yt_info.update({"author":yt.author})
            yt_info.update({"caption_tracks":yt.caption_tracks})
            yt_info.update({"captions":yt.captions})
            yt_info.update({"channel_id":yt.channel_id})
            yt_info.update({"channel_url":yt.channel_url})
            yt_info.update({"chapters":yt.chapters})
            yt_info.update({"description":yt.description})
            yt_info.update({"length":yt.length})
            yt_info.update({"keywords":yt.keywords})
            yt_info.update({"publish_date":yt.publish_date})
            yt_info.update({"rating":yt.rating})
            yt_info.update({"key_moments":yt.key_moments})
            yt_info.update({"metadata":yt.metadata})
            yt_info.update({"views":yt.views})
            yt_info.update({"vid_info":yt.vid_info})
        return yt_info    

```

__________
**Expected behavior**
It used to return all info. 
 

__________
**Screenshots**

__________
**Desktop (please complete the following information):**
 - OS: Windows 10 pro
 - Python Version 3.9.13
 - Pytubefix Version 7.1rc2

__________
**Additional context**

**Comment by JuanBindez** [comment]
try the lastest version

**Comment by fedetony** [comment]
Ok :) version 7.1.3 works fine, was using 7.1rc2 because of the video unavailable error.

**Comment by fedetony** [comment]
All fixed in 7.1.3
